### ISSUE TITLE
Error bad pointer with wails

### ISSUE THREAD

**Issue by thomasdqr** [fcjr/geticon-go#2]
Hi !

Your library is amazing ! Thanks for the share, it works really well 👌 
I have an issue tho when using it with [wails](https://wails.io/)

When I try to execute the getIcon function from my frontend app, it throw a bad pointer error 🤔 

```
DEB | [ExternalAssetHandler] Loading 'http://localhost:5173/wailsjs/go/main/App.js'
runtime: bad pointer in frame github.com/fcjr/geticon/internal/winapi.EnumResourceNamesA at 0xc0003369f8: 0xe
fatal error: invalid pointer found on stack

runtime stack:
runtime.throw({0x10febf5?, 0x100000000?})
        C:/Program Files/Go/src/runtime/panic.go:1077 +0x65 fp=0x56899ff248 sp=0x56899ff218 pc=0x873125
runtime.adjustpointers(0xc0003369e0, 0x56899ff358, 0x0?, {0x56899ff378?, 0x89516b?})
        C:/Program Files/Go/src/runtime/stack.go:627 +0x285 fp=0x56899ff2d8 sp=0x56899ff248 pc=0x888925
runtime.adjustframe(0x56899ff4f8, 0x56899ff3d8)
```

When I try to run the function at the load of my app directly from the main.go file, it works well, but as soon as I try to run it from my app, I have the error above 😞 

It is easily replicable by setting up a template project using wails, whatever the frontend framework chosen (I tried react, and vanilla js) and try to call a simple getIcon function inside the default Greet function for example, you should get the same error as mine

Any help would be immensely appreciated 🙏

**Comment by fcjr** [comment]
Hey @thomasdqr glad you are enjoying the library and sorry your having this issue! I will try to reproduce, in the meantime, what application are you trying to get the icon of and which method (path, or pid)?

**Comment by thomasdqr** [comment]
Hi ! I am using the chrome icon with the path : `C:\Program Files\Google\Chrome\Application\chrome.exe` It works fine when I run it at the start of the main.go script, but as soon as the app is fully started and I try to call the function again from the frontend, I have the error above.
If you need any other information / examples don't hesitate to ask 🙏

**Comment by thomasdqr** [comment]
In the meantime, I've rebuilt my app using Tauri, that uses Rust instead of Go for the backend (I prefer golang but it was the only way to make the icon retrieving working) and it uses this plugin : https://github.com/uriegel/systemicons for the retrieving, I don't know if it can help you see the winapi implementation made there 🤔

**Comment by fcjr** [comment]
Took a quick look and was able to reproduce when running as `wail dev` though it works perfectly when built to its own .exe.  I haven't figured out exactly what's happening as I am not too familiar with the inner workings of wails, but at a cursory look it may be related to https://github.com/golang/go/issues/22192 which would only effect `wails dev` but isn't something that could be solved in this library.  Might be worth raising an issue with @leaanthony over at https://github.com/wailsapp/wails.

**Comment by fcjr** [comment]
Hey @thomasdqr I was able to reproduce -- looks like it was an issue MakeIntResource.  Released a fix as v0.1.2, let me know if it works for you!

**Comment by thomasdqr** [comment]
Wow thank you ! Works like a charm 👌
